packages/core/src/Stores/ui-store.js

```diff
@@ -26,7 +26,8 @@ export default class UIStore extends BaseStore {
     settings_extension = undefined;
     notification_messages_ui = undefined;
 
-    is_dark_mode_on = window?.matchMedia?.('(prefers-color-scheme: dark)').matches && isMobile();
+    is_dark_mode_on = false;
```

Comment:
@arshad-rao-deriv looks like this code was added for the users which prefer dark theme, but now we made decision to switch to the light theme for all :) it's strange. But ok, it should work for your task )